DorisProject/src/app/pages/modal/modal.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ModalComponent } from './modal.component';
+
+describe('ModalComponent', () => {
+  let component: ModalComponent;
+  let fixture: ComponentFixture<ModalComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ ModalComponent ]
```

Comment:
Codacy has a fix for the issue: [Replace `·ModalComponent·]` with `ModalComponent],`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
      declarations: [ModalComponent],
```